### ISSUE TITLE
Tick marks

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/genepanel.iobio#63]
We should fix the number of tick marks and numbers displayed on the axes to a fixed number. In the image below (cardiomyopathy), we don't try and fill the y axis with tick marks, just add them at a reasonable spacing. Do the same thing on the x axis.

<img width="460" alt="screen shot 2018-04-02 at 3 35 42 pm" src="https://user-images.githubusercontent.com/523111/38212379-9318ea18-368b-11e8-8d30-cd0c8a8146e9.png">

**Comment by adityaekawade** [comment]
I have made changes to the X axis, so it doesn't get cluttered even with large data. 

![Screen Shot 2018-04-02 at 3.50.42 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a68ffc37453d30014ef4f0d/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77276454f16b7191a37bf1172b4f5b5d5ecc3c4a4356bcbbd671f3e57862804e0ebcd8380b7dcf6f14faee824dba01ba21d94d86e7b48818d2f42f67b6d5db1063474f4dcbed5727a19141c95df449e64d53a110366ae6702431.png)


Do you want the numbers on the Y axis to be constant (1, 2, 4, 16, 32, 128, 256, 1024)  like it is in Gene Membership? The top number on the Y axis usually varies between 50- 1400 for different search terms.

**Comment by AlistairNWard** [comment]
No, I think this should be dynamically scaled to the incoming data. How you have it above looks great.
